### PR TITLE
+fixes

### PR DESCRIPTION
**Pull request by memmed8ov**

### CODE REVIEW

File: 2/react-app/src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,49 @@
...
-    </div>
-  )
+import { BrowserRouter as Router, Routes, Route, useNavigate } from 'react-router-dom'
+
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Button from '@mui/material/Button';
+
+ import {Countries} from './rout/country/country-page';
```
Comment (tislib) [inline 2/react-app/src/App.jsx]: no space is needed before import command

File: 2/react-app/src/rout/city/CityAdd.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            <br />
+            <input placeholder='code' style={{ marginLeft: '50px' }} type="text" value={code} onChange={e => setCode((e.target.value))} />
+            <br />
+            <br />
+            <button onClick={() => {
+                let object = {
+                    "properties": {
+                        "description": code,
+                        "name": name
+
```
Comment (tislib) [inline 2/react-app/src/rout/city/CityAdd.jsx]: remove this line

**Review by tislib — CHANGES_REQUESTED** [review]

**Comment by memmed8ov** [comment]
uyuuuu hocaaaaaa

File: 2/react-app/src/rout/city/CityAdd.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                    }
+                }
+
+                axios.post('http://tiswork.tisserv.net:8008/country', object).then(resp => {
+                    navigate('..')
+                })
+
+
```
Comment (tislib) [inline 2/react-app/src/rout/city/CityAdd.jsx]: [OPTIONAL] There are useless new lines (multiple new lines is not good practice for code formatting)

File: 2/react-app/src/rout/city/CityAdd.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            <input placeholder='name' style={{ marginLeft: '50px' }} type="text" value={name} onChange={e => setName((e.target.value))} />
+            <br />
+            <br />
+            <input placeholder='code' style={{ marginLeft: '50px' }} type="text" value={code} onChange={e => setCode((e.target.value))} />
+            <br />
+            <br />
+            <button onClick={() => {
+                let object = {
+                    "properties": {
+                        "description": code,
```
Comment (tislib) [inline 2/react-app/src/rout/city/CityAdd.jsx:22]: "description": code

why you have named field for description as code.
Please use correct name on field code (it should be description)

File: 2/react-app/src/rout/city/CityList.jsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react'
+import axios from 'axios'
+import { useNavigate } from 'react-router-dom'
+
+export function CityList() {
+    const navigate = useNavigate()
+    const [data, setData] = useState([])
+
+
+    function allcode() {
```
Comment (tislib) [inline 2/react-app/src/rout/city/CityList.jsx]: function name is incorrect, 
Function name should be related to what it is doing.

allcode => loadCountryList OR load

Comment (tislib) [reply 2/react-app/src/rout/city/CityList.jsx]: [Important]

File: 2/react-app/src/rout/city/CityUpdate.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+import { Navigate, useNavigate, useParams } from 'react-router-dom'
+
+export function CityUpdate() {
+    const navigate = useNavigate()
+    const params = useParams()
+
+    const itemId = params.id
+
+    const [nameupdate, setNameUpdate] = useState('')
+    const [codeupdate, setCodeUpdate] = useState('')
```
Comment (tislib) [inline 2/react-app/src/rout/city/CityUpdate.jsx]: Same here, code <=> description, the variable name is not related to what it does. please rename it to description

**Review by tislib — CHANGES_REQUESTED** [review]

File: 2/react-app/src/rout/city/CityAdd.jsx (hunk 4 above)
Comment (memmed8ov) [reply 2/react-app/src/rout/city/CityAdd.jsx:22]: because country codes were requested.i called the code.
I named it "description" in our previous study

**Review by tislib — APPROVED** [review]

**Pull request merged**